### PR TITLE
Moving ROS communication to external file

### PR DESCRIPTION
**Pull request by kelvinkoon**
### Description

ROS actions that communicate using Rosbridge have been moved into a util file to make the code better organized. This will be the basis for communicating with the ROS parameter server that includes a function to send request to service.

### Testing Done

Running the web application with Rosbridge yields the same result where console displays ROS messages. Nothing has been broken as a result which means the changes were successful.

### Resolved Issues

Working towards resolving #392 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
- [x] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom`
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/corner_kick/src/utils/ros/index.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    const topic = new Topic({
+        messageType,
+        name,
+        ros,
+    });
+    topic.unsubscribe(callback);
+};
+
+/**
+ * Update ROS Param server by sending request to service
```
Comment (qgolsteyn) [inline src/corner_kick/src/utils/ros/index.ts:76]: More detailed comment describing each param (using `@params`, see JSDoc format) would be nice

File: src/corner_kick/src/utils/ros/index.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    const topic = new Topic({
+        messageType,
+        name,
+        ros,
+    });
+    topic.subscribe(callback);
+};
+
+/**
+ * Unsubscribe from ROS Topic
```
Comment (qgolsteyn) [inline src/corner_kick/src/utils/ros/index.ts:57]: More detailed comment describing each param (using @params, see JSDoc format) would be nice

File: src/corner_kick/src/utils/ros/index.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+/**
+ * Disconnect from ROS
+ */
+export const stop = () => {
+    ros.close();
+};
+
+/**
+ * Subscribe to ROS Topic
```
Comment (qgolsteyn) [inline src/corner_kick/src/utils/ros/index.ts:38]: More detailed comment describing each param (using @params, see JSDoc format) would be nice

File: src/corner_kick/src/utils/ros/index.ts [hunk 4]
```diff
@@ -0,0 +1,94 @@
+/*
+ * This file contains ROS methods to get and set values in the ROS Param Server
+ */
+
+import { Message, Ros, Service, ServiceRequest, Topic } from 'roslib';
+
+const ros: Ros = new Ros({});
+
+/**
+ * Connect to ROS
```
Comment (qgolsteyn) [inline src/corner_kick/src/utils/ros/index.ts:10]: More detailed comment describing each param (using @params, see JSDoc format) would be nice. Also indicate under what condition the Promise is accepted vs rejected.

**Review by qgolsteyn — CHANGES_REQUESTED** [review]
Just needs a few more comments, then it's all good for launch :rocket:

File: src/corner_kick/src/utils/ros/index.ts (hunk 1 above)
Comment (kelvinkoon) [reply src/corner_kick/src/utils/ros/index.ts:76]: Updated to describe each param.

File: src/corner_kick/src/utils/ros/index.ts (hunk 3 above)
Comment (kelvinkoon) [reply src/corner_kick/src/utils/ros/index.ts:38]: Updated to describe each param.

File: src/corner_kick/src/utils/ros/index.ts (hunk 4 above)
Comment (kelvinkoon) [reply src/corner_kick/src/utils/ros/index.ts:10]: Updated to describe each param.

**Review by qgolsteyn — APPROVED** [review]
LGTM

**Pull request merged**